src/components/pages/dashboard-pages/EditLessonForm.jsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import FormInput from '../../common/FormInput';
+
+const EditLessonForm = ({
+  formData,
+  formErrors,
+  handleChange,
+  handleSubmit,
+}) => {
+  const navigate = useNavigate();
+  const handleCancel = () => {
+    navigate(-1);
+  };
+
+  return (
+    <div className="w-full flex flex-col items-center p-4">
+      <h1 className="text-black font-semibold text-xl sm:text-2xl font-spartan mb-4">
+        Edit Lesson
+      </h1>
+      <form className="flex flex-col w-full max-w-xl" onSubmit={handleSubmit}>
+        {formErrors.form && (
+          <p className="text-red font-spartan mb-2">{formErrors.form}</p>
+        )}
+        <div className="flex flex-col gap-4">
+          {formErrors.classId && (
+            <p className="text-red text-sm font-spartan">
+              {formErrors.classId}
+            </p>
+          )}
+          <FormInput
+            placeholder=" "
+            name="lessonTitle"
+            value={formData.lessonTitle}
+            onChange={handleChange}
+            min="2"
+            max="100"
+          >
+            Lesson Title
+          </FormInput>
+          {formErrors.lessonTitle && (
+            <p className="text-red text-sm font-spartan">
+              {formErrors.lessonTitle}
+            </p>
+          )}
+          <div className="flex gap-4 items-center">
+            <p className="w-1/4">Date & Time:</p>
+            <div className="flex gap-4 w-3/4">
+              <FormInput
+                type="date"
+                name="lessonSchedule.date"
+                value={formData.lessonSchedule.date}
+                onChange={handleChange}
+                min={new Date().toISOString().split('T')[0]}
+                className="w-1/2"
+              />
+              <FormInput
+                type="time"
+                name="lessonSchedule.startTime"
+                value={formData.lessonSchedule.startTime}
+                onChange={handleChange}
+                className="w-1/2"
+              />
+            </div>
+          </div>
+          {formErrors['lessonSchedule.date'] && (
+            <p className="text-red text-sm font-spartan">
+              {formErrors['lessonSchedule.date']}
+            </p>
+          )}
+          {formErrors['lessonSchedule.startTime'] && (
+            <p className="text-red text-sm font-spartan">
+              {formErrors['lessonSchedule.startTime']}
+            </p>
+          )}
+          <div className="flex gap-4 my-4">
+            <p className="w-1/4">Type:</p>
+            <div className="flex flex-col gap-2 w-3/4">
+              <label className="flex gap-1 items-center">
+                <input
+                  type="radio"
+                  name="type"
+                  value="online"
+                  checked={formData.type === 'online'}
+                  className="w-4 h-4 accent-lightGreen focus:darkGreen"
```

Comment:
nit picky: To clean things up you can extract these repetitive classes into a constant and pass that along. Keep this in mind for future code bases to clean things up.